db/pre-re/skill_db.conf

```diff
@@ -24040,16 +24029,16 @@ skill_db: (
 	}
 	AttackType: "Weapon"
 	SkillData1: {
-		Lv1: 3500
-		Lv2: 4000
-		Lv3: 4500
-		Lv4: 5000
-		Lv5: 5500
-		Lv6: 6000
-		Lv7: 6500
-		Lv8: 7000
-		Lv9: 7500
-		Lv10: 8000
+		Lv1: 5500
+		Lv2: 6000
+		Lv3: 6500
+		Lv4: 7000
+		Lv5: 7500
```

Comment:
@Smokeyxyz as said I can't remember exactly but this was talked with haru and Ema as was declined, so I'd suggest to wait for their input here